tox.ini

```diff
@@ -48,8 +48,5 @@ commands =
     flake8 {toxinidir}/vyper {toxinidir}/tests {toxinidir}/scripts
     flake8 {toxinidir}/bin/vyper {toxinidir}/bin/vyper-lll {toxinidir}/bin/vyper-serve
     mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs -p vyper
-    mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs bin/vyper
-    mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs bin/vyper-lll
-    mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs bin/vyper-serve
     mypy --follow-imports=silent --ignore-missing-imports --disallow-incomplete-defs scripts
-    isort --recursive --check-only --diff {toxinidir}/vyper {toxinidir}/tests {toxinidir}/bin {toxinidir}/scripts
+    isort --recursive --check-only --diff {toxinidir}/vyper {toxinidir}/tests {toxinidir}/scripts
```

Comment:
@jacqueswww is the stuff in `scripts/` still relevant?

Reply:
for time being yes.